node/sc/multi_bridge_test.go

```diff
@@ -0,0 +1,356 @@
+
+	cBridge, err := info.b.CounterpartBridge(nil)
+	assert.NoError(t, err)
+	assert.Equal(t, dummy, cBridge)
+}
+
+// TestRegisterDeregisterToken checks the following:
+// - the bridge contract method RegisterToken and DeregisterToken.
+func TestRegisterDeregisterToken(t *testing.T) {
+	info := prepareMultiBridgeTest(t)
+	dummy1 := common.Address{10}
+	dummy2 := common.Address{20}
+
+	opts := &bind.TransactOpts{From: info.acc.From, Signer: info.acc.Signer, GasLimit: gasLimit}
+	tx, err := info.b.RegisterToken(opts, dummy1, dummy2)
+	assert.NoError(t, err)
+	info.sim.Commit()
+	assert.Nil(t, bind.CheckWaitMined(info.sim, tx))
+
+	res, err := info.b.AllowedTokens(nil, dummy1)
+	assert.NoError(t, err)
+	assert.Equal(t, dummy2, res)
+
+	opts = &bind.TransactOpts{From: info.acc.From, Signer: info.acc.Signer, GasLimit: gasLimit}
+	tx, err = info.b.DeregisterToken(opts, dummy1)
+	assert.NoError(t, err)
+	info.sim.Commit()
+	assert.Nil(t, bind.CheckWaitMined(info.sim, tx))
+
+	res, err = info.b.AllowedTokens(nil, dummy1)
+	assert.NoError(t, err)
+	assert.Equal(t, common.Address{0}, res)
+}
+
+// TestOperatorThreshold checks the following:
+// - the bridge contract method SetOperatorThreshold
+func TestOperatorThreshold(t *testing.T) {
+	info := prepareMultiBridgeTest(t)
+	const vtThreshold = uint8(128)
+	const confThreshold = uint8(255)
+
+	opts := &bind.TransactOpts{From: info.acc.From, Signer: info.acc.Signer, GasLimit: gasLimit}
+	tx, err := info.b.SetOperatorThreshold(opts, voteTypeValueTransfer, vtThreshold)
+	assert.NoError(t, err)
+	info.sim.Commit()
+	assert.Nil(t, bind.CheckWaitMined(info.sim, tx))
+
+	opts = &bind.TransactOpts{From: info.acc.From, Signer: info.acc.Signer, GasLimit: gasLimit}
+	tx, err = info.b.SetOperatorThreshold(opts, voteTypeConfiguration, confThreshold)
+	assert.NoError(t, err)
+	info.sim.Commit()
+	assert.Nil(t, bind.CheckWaitMined(info.sim, tx))
+
+	res, err := info.b.OperatorThresholds(nil, voteTypeValueTransfer)
+	assert.NoError(t, err)
+	assert.Equal(t, vtThreshold, res)
+
+	res, err = info.b.OperatorThresholds(nil, voteTypeConfiguration)
+	assert.NoError(t, err)
+	assert.Equal(t, confThreshold, res)
+}
+
+// TestMultiBridgeKLAYTransfer1 checks the following:
+// - successful value transfer with proper transaction counts.
+func TestMultiBridgeKLAYTransfer1(t *testing.T) {
+	info := prepareMultiBridgeEventTest(t)
+	acc := info.accounts[0]
+	to := common.Address{100}
+
+	opts := &bind.TransactOpts{From: acc.From, Signer: acc.Signer, GasLimit: gasLimit}
+	tx, err := info.b.SetOperatorThreshold(opts, voteTypeValueTransfer, 2)
+	assert.NoError(t, err)
+	info.sim.Commit()
+	assert.Nil(t, bind.CheckWaitMined(info.sim, tx))
+
+	nonceOffset := uint64(17)
+	sentNonce := nonceOffset
+	transferAmount := uint64(100)
+	sentBlockNumber := uint64(100000)
+
+	tx = SendHandleKLAYTransfer(info.b, acc, to, transferAmount, sentNonce, sentBlockNumber, t)
+	info.sim.Commit()
+	assert.NoError(t, bind.CheckWaitMined(info.sim, tx))
+
```

Comment:
If possible, we need to check the vote count and if it is executed or not in next PR.